e2e/routes.spec.ts

```diff
@@ -0,0 +1,147 @@
+const routes = [
+    "/", "/activity", "/feedback", "/gift", "/receive", "/redshift", "/scanner", "/send", "/swap", "/settings",
+]
+
+const settingsRoutes = [
+    "/admin", "/backup", "/channels", "/connections", "/currency", "/emergencykit", "/encrypt", "/gift", "/lnurlauth", "/plus", "/restore", "/servers", "/syncnostrcontacts"
+]
+
+const settingsRoutesPrefixed = settingsRoutes.map((route) => {
+    return "/settings" + route
+});
+
+const allRoutes = routes.concat(settingsRoutesPrefixed);
+
+// Create a JS Map of all routes so we can check them off one by one
+const checklist = new Map();
+allRoutes.forEach((route) => {
+    checklist.set(route, false);
+});
+
+import { test, expect, Page } from "@playwright/test";
+
+// Only works if there's a link to the route on the page
+async function checkRoute(page: Page, route: string, expectedHeader: string, checklist: Map<string, boolean>) {
+    await page.locator(`a[href='${route}']`).first().click();
+    await expect(page.locator("h1").first()).toHaveText(expectedHeader);
+    checklist.set(route, true);
+}
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("http://localhost:3420/");
+});
+
+test("visit each route", async ({ page }) => {
+    // Start on the home page
+        // Expect a title "to contain" a substring.
+        await expect(page).toHaveTitle(/Mutiny Wallet/);
+
+        // Wait for an element matching the selector to appear in DOM.
+        await page.waitForSelector("text=0 SATS");
+    
+        console.log("Page loaded.");
+    
+        // Wait for a while just to make sure we can load everything
+        await page.waitForTimeout(1000);
+
+        checklist.set("/", true);
+
+        await checkRoute(page, "/activity", "Activity", checklist);
+        await page.goBack();
+
+        // Navigate to settings
+        await checkRoute(page, "/settings", "Settings", checklist);
+
+        // Mutiny+
+        await checkRoute(page, "/settings/plus", "Mutiny+", checklist);
+        await page.goBack();
+
+        // Lightning Channels
+        await checkRoute(page, "/settings/channels", "Lightning Channels", checklist);
+        await page.goBack();
+
+        // Backup
+        await checkRoute(page, "/settings/backup", "Backup", checklist);
+        await page.goBack();
+
+        // Restore
+        await checkRoute(page, "/settings/restore", "Restore", checklist);
+        await page.goBack();
+
+        // Currency
+        await checkRoute(page, "/settings/currency", "Currency", checklist);
+        await page.goBack();
+
+        // Servers
+        await checkRoute(page, "/settings/servers", "Servers", checklist);
+        await page.goBack();
+
+        // Connections
+        await checkRoute(page, "/settings/connections", "Wallet Connections", checklist);
```

Comment:
Can you test the wallet connections page with query parameters too? Like you did with the gift one below.

Reply:
added a test for wallet connections with params